solutions/swb-reference/integration-tests/support/resources/environments/environment.ts

```diff
@@ -13,6 +14,40 @@ export default class Environment extends Resource {
     super(clientSession, 'environment', id, parentApi);
   }
 
+  public async pollEnvironment(
```

Comment:
I was thinking on moving this to Resource but I wasn't sure how useful polling would be useful for other entities.
Any thoughts @nguyen102  @SanketD92

Reply:
Environment is the only resource currently needing a polling mechanism, let's hold off on moving this logic to the base class.

Reply:
rename method to `poll()`